src/api/server.go

```diff
@@ -12,6 +12,9 @@ func Server() (e *echo.Echo) {
 
 	// Allow browser clients to use the API
 	e.Use(middleware.CORS())
+	e.Use(middleware.LoggerWithConfig(middleware.LoggerConfig{
```

Comment:
Rather than duplicating this everywhere we should probably define it in our middleware package